src/Microsoft.AspNetCore.Build/targets/makefile.proj

```diff
@@ -1,13 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" InitialTargets="Initialize" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-    <PropertyGroup>
-        <!-- Product-specific properties -->
-        <DefaultAuthors>Microsoft Open Technologies, Inc.</DefaultAuthors>
-        <ProductStartDate>2015-01-01</ProductStartDate>
-        <BuildQuality>alpha</BuildQuality>
-        <BuildVersion>1.0.1</BuildVersion>
-    </PropertyGroup>
-
+    <Import Project="$(MSBuildThisFileDirectory)\Common.settings" />
```

Comment:
Also, can you rename the file to `Common.props`? Generally there are `.targets` and `.props` files in MSBuild, so I'd like to be consistent. I have occasionally seen `.settings`, but I think `.props` is more common :)